contracts/src/river/registry/facets/stream/StreamRegistry.sol

```diff
@@ -285,70 +281,4 @@ contract StreamRegistry is IStreamRegistry, RegistryModifiers {
 
     return (streams, stop >= streamCount);
   }
-
-  function getStreams(
-    bytes32[] calldata streamIds
-  ) external view returns (uint256 foundCount, StreamWithId[] memory) {
-    uint256 streamCount = streamIds.length;
-    StreamWithId[] memory streams = new StreamWithId[](streamCount);
-    for (uint256 i = 0; i < streamCount; ++i) {
-      bytes32 streamId = streamIds[i];
-      Stream storage stream = ds.streamById[streamId];
-      if (stream.nodes.length == 0) continue;
-      streams[foundCount++] = StreamWithId({id: streamId, stream: stream});
-    }
-    return (foundCount, streams);
-  }
-
-  function getStreamsOnNode(
```

Comment:
I thought `getStreamsOnNode` will be used.

Reply:
nope

Reply:
getStreamCountOnNode is

Reply:
Isn't it supposed to make node sync faster?

Reply:
I'm removing all functions we agreed on removing, if we need add new ones that are optimized we will add them back